src/site/pages/digital/src/containers/SelectionPopup/modules/InputCountModule.tsx

```diff
@@ -1,20 +1,16 @@
 import {GroupAction} from "core/actions/GroupAction";
 
-import {ANDGate, ORGate, XORGate} from "digital/models/ioobjects";
-import {NANDGate} from "digital/models/ioobjects/gates/ANDGate";
-import {NORGate} from "digital/models/ioobjects/gates/ORGate";
-import {XNORGate} from "digital/models/ioobjects/gates/XORGate";
+import {Gate} from "digital/models/ioobjects/gates/Gate";
 import {Mux} from "digital/models/ioobjects/other/Mux";
 
 import {InputPortChangeAction} from "digital/actions/ports/InputPortChangeAction";
 
 import {CreateModule, ModuleConfig, PopupModule} from "shared/containers/SelectionPopup/modules/Module";
 
 
-const Config: ModuleConfig<[ANDGate, NANDGate, ORGate,
-                            NORGate, XORGate, XNORGate], number> = {
-    types: [ANDGate, NANDGate, ORGate,
-            NORGate, XORGate, XNORGate],
+
```

Comment:
nit: only two spaces between last import and start of code, so this empty line is unnecessary

Reply:
This still needs to be addressed, but looks good otherwise